packages/react-native/src/theme/tokens/index.ts

```diff
@@ -0,0 +1,53 @@
+import { components } from './components';
+import { ComponentStyles } from '../types';
+import { ColorValue } from '@aws-amplify/ui';
+
+interface BaseTokens {
+  colors: Partial<Colors>;
+}
+
+export interface Tokens extends BaseTokens {
+  components: ComponentStyles;
+}
+
+// TODO: remove color types and values and use the output generated by style-dictionary with correct types
+
+declare type ScaleKeys = 10 | 20 | 40 | 60 | 80 | 90 | 100;
+type ColorScale = {
+  [key in ScaleKeys]: ColorValue;
+};
+
+type Colors = {
+  brand: {
+    primary: ColorScale;
+    secondary: ColorScale;
+  };
+};
+
+export const colors: Partial<Colors> = {
```

Comment:
Why is this type with a `Partial`? Don't we expect these values to be fully provided?

Reply:
Just wanted to bring in a couple colors for example purposes. This will be changed once we generate the comp. agnostic tokens.